### PR TITLE
add list command(get file list, but not download file body)

### PR DESCRIPTION
**Pull request by itn3000**
for ensuring what files are downloaded before real download

### CODE REVIEW

File: cmd_list.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	for {
+		file, err := slist.Recv()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			break
+		}
+		if runtime.GOOS != "windows" {
+			os.Stdout.WriteString(fmt.Sprintf("\"%v\",\"%v\",\"%v\",\"%v\"\n",
```
Comment (mattn) [inline cmd_list.go]: Just call `fmt.Printf(...)`.

Comment (mattn) [reply cmd_list.go]: Use `%q` instead of `"%s"`.

File: cmd_list.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+)
+
+func listFiles(ctx context.Context, client proto.FileTransferServiceClient) error {
+	slist, err := client.ListFiles(ctx, new(proto.ListRequestType))
+	if err != nil {
+		return err
+	}
+	os.Stdout.WriteString("name,size,mode,modtime\n")
```
Comment (mattn) [inline cmd_list.go]: `fmt.Println(...)`

**Comment by itn3000** [comment]
thank you for reviewing, I fixed.

File: cmd_list.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+		return err
+	}
+	fmt.Println("name,size,mode,modtime")
+	for {
+		file, err := slist.Recv()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			break
```
Comment (mattn) [inline cmd_list.go]: remove needless this if-condition.

File: cmd_list.go [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+		}
+		if runtime.GOOS != "windows" {
+			fmt.Printf("%q,\"%v\",\"%v\",\"%v\"\n",
+				file.Name, file.Size, os.FileMode(file.Mode), time.Unix(file.ModTime.Seconds, 0).Format(time.RFC3339))
+		} else {
+			fmt.Printf("%q,\"%v\",\"%o\",\"%v\"\n",
+				file.Name, file.Size, file.Mode, time.Unix(file.ModTime.Seconds, 0).Format(time.RFC3339))
+		}
```
Comment (mattn) [inline cmd_list.go]: Why this part is separeted for Windows?

Comment (itn3000) [reply cmd_list.go]: I misunderstood that os.FileMode should not use in windows.
I will remove the if block.

**Comment by mattn** [comment]
LGTM. Thank you.

**Pull request merged**